x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/langident/LangIdentNeuralNetwork.java

```diff
@@ -156,11 +146,6 @@ public TargetType targetType() {
         return TargetType.CLASSIFICATION;
     }
 
-    @Override
-    public List<String> classificationLabels() {
-        return LANGUAGE_NAMES;
```

Comment:
Consider adding `LANGUAGE_NAMES` to `toXContent` and the parser. `Tree` includes the labels in its xcontent representation